charts/ctlog/README.md

```diff
@@ -76,6 +79,7 @@ Certificate Log
 | server.podAnnotations."prometheus.io/scrape" | string | `"true"` |  |
 | server.portHTTP | int | `6962` |  |
 | server.portHTTPMetrics | int | `6963` |  |
+| server.resources | string | `""` |  |
```

Comment:
I believe you are missing this field in the ctlog-deployment.

Reply:
ctlog-deployment already have resource defined but it was missing an entry in the README
https://github.com/sigstore/helm-charts/blob/main/charts/ctlog/templates/ctlog-deployment.yaml#L37